src/BionetExchange.sol

```diff
@@ -76,6 +76,10 @@ contract BionetExchange is IBionetExchange, ReentrancyGuard {
         require(_caller == offer.seller, "Exchange: caller must be the seller");
         offer.voided = true;
 
+        // make deposit available to withdraw after void
+        uint256 deposit = FundsLib.calculateFee(offer.price, CANCEL_REVOKE_FEE);
```

Comment:
This should probably be handled with "finalizeCommittment" to be consistent with others, like "cancel".  And is there a fee?   Note:  this is all changing dramatically very soon

Reply:
It could be. One of the params of finalizeCommittment is _exchangeId. Technically, until someone commits to an offer, it only has an offerId. I could pass that into finalizeCommittment, but that could be confusing. There wouldn't be a fee because the seller is voiding the offer before anyone has committed.